render/board.go

```diff
@@ -37,22 +37,43 @@ func (b *Board) setSquare(p *engine.Point, s BoardSquare) {
 }
 
 func (b *Board) getDirection(p engine.Point, nP engine.Point) string {
-	d := fmt.Sprintf("%d,%d", nP.X-p.X, p.Y-nP.Y)
-	switch d {
-	case "1,0":
+	// handle cases where we aren't wrapping around the board
+	if p.X+1 == nP.X {
```

Comment:
Note that it's considered idiomatic Go style to not use `else` / `else if` statements when `if` statements return